Comp2022/src/main/include/subsystems/Climber.h

```diff
@@ -88,14 +90,21 @@ class Climber : public frc2::SubsystemBase
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
+    enum
+    {                         // Climber subsystem movement states
+        NOCHANGE_HEIGHT = -1, // No change in climber height--maintain current position
+        STOW_HEIGHT = 0,      // Move to stow height
+        EXTEND_L2_HEIGHT = 1, // Move to extend to L2 height
+        ROTATE_L3_HEIGHT = 2, // Move to rotate to L3 height
+        EXTEND_L3_HEIGHT = 3, // Move to extend to L3 height
+    };
+
     void Initialize(void);
     void FaultDump(void);
     void MoveClimberWithJoysticks(frc::XboxController *operatorController);
     void SetGateHook(bool hookClosed);
     void SetClimberStopped(void);
 
-    const bool CL_BRAKE_LOCKED = false;
-    const bool CL_BRAKE_UNLOCKED = true;

```

Comment:
I liked having names for these values - I never remember what `false` means and what `true` means

Can we bring these back?

Reply:
We don't have a pneumatic brake on the climber anymore, so I decided to take it out

Reply:
Ohh that makes sense - I was mix-up-ing this with the gate latch constants :+1: